docs/installation/local.md

```diff
@@ -30,6 +30,11 @@ The community provides packages of Loki for openSUSE Linux. To install:
 
 ### Building
 
+Enable go modules
```

Comment:
I think this is required only for go < 1.13 ?

Reply:
@cyriltovena yeah, my bad

Reply:
You can keep the PR but add a comment to explain this is required only for version below 1.13.

Reply:
@cyriltovena I used `go1.12.8` when tried to install Loki, so probably yes, but as far as there's `go1.13+` in requirements I'm not sure this PR is actual, although I can reopen it if you think it's still worth to add this line to README

Reply:
All I'm saying is, I'm fine to have this if you explain when exactly it is required and when it is not.